components/docs-chef-io/content/automate/node_bootstarp_ha.md

```diff
@@ -0,0 +1,148 @@
++++
+title = "Node Bootstrapping"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Node Bootstrapping"
+    identifier = "automate/configuring_automate/node_bootstarp_ha.md Node Bootstrapping"
+    parent = "automate/configuring_automate"
+    weight = 90
++++
+
+A node is any physical, virtual, or cloud device configured and maintained by an instance of Chef Infra Client. Bootstrapping installs Chef Infra Client on a target system so it can run as a client and sets the node up to communicate with a Chef Infra Server.
+
+To bootstrap a node, run `knife bootstrap` in the workstation.
+
+## Knife Bootstrap
+
+The knife bootstrap command is a common way to install Chef Infra Client on a node. The default for this approach assumes that a node can access the Chef website so that it may download the Chef Infra Client package from that location.
+
+The Chef Infra Client installer will detect the version of the operating system and then install the appropriate Chef Infra Client version using a single command to install Chef Infra Client and all of its dependencies, including an embedded version of Ruby, OpenSSL, parsers, libraries, and command-line utilities.
+
+The Chef Infra Client installer puts everything into a unique directory (/opt/chef/) so that Chef Infra Client will not interfere with other applications that may be running on the target machine. Once installed, Chef Infra Client requires a few more configuration steps before performing its first Chef Infra Client run on a node.
+
+## Bootstrap a Node
+
+### Create Users and Organization
+
+1. ssh in the bastion host and follow the below steps.
+
+2. ssh in chef_server instance.
+
+`sudo chef-automate ssh --hostname chef_server`
```

Comment:
If it is going under HA then this command is applicable, otherwise we can mention to go automate standalone machine itself

Reply:
It is going under HA section